extra/release_tool.py

```diff
@@ -2749,6 +2749,20 @@ def do_set_version_to(args):
         integration_dir(), repo, args.version, git_tag=args.version
     )
 
+    # Update extra files used in integration tests
+    set_docker_compose_version_to(
+        os.path.join(integration_dir(), "extra", "mtls"),
+        repo,
+        args.version,
+        git_tag=args.version,
+    )
+    set_docker_compose_version_to(
+        os.path.join(integration_dir(), "extra", "failover-testing"),
+        repo,
+        args.version,
+        git_tag=args.version,
+    )
```

Comment:
I don't have a better suggestion at the moment, but we should think about a better way to do this. It's not very maintainable long term.

Reply:
I first thought about making `set_docker_compose_version_to` recursively look for all `yml` files, so that here a single call would do, but the challenge is that there are some files (like the compatibility tests for the client) that we don't want to modify, and then we would need to handle that as a exception of some sort.

Conclusion: I don't have a good suggestion neither :)